examples/siro_sandbox/sandbox_app.py

```diff
@@ -93,23 +96,54 @@ def set_debug_line_render(self, debug_line_render):
     def get_sim(self) -> Any:
         return self.env.task._sim
 
+    def draw_nav_hint_from_agent(self, end_pos, color):
+        agent_idx = 0
+        art_obj = (
+            self.get_sim().agents_mgr[agent_idx].articulated_agent.sim_obj
+        )
+        agent_pos = art_obj.transformation.translation
+        # todo: get forward_dir from FPS camera yaw, not art_obj.transformation
+        # (the problem with art_obj.transformation is that it includes a "wobble"
+        # introduced by the walk animation)
+        forward_dir = art_obj.transformation.transform_vector(
+            mn.Vector3(0, 0, 1)
+        )
+        forward_dir[1] = 0
+
+        self.draw_nav_hint(
+            agent_pos, forward_dir, end_pos, color, self._viz_anim_fraction
+        )
+
     def visualize_task(self):
         sim = self.get_sim()
         idxs, goal_pos = sim.get_targets()
         scene_pos = sim.get_scene_pos()
         target_pos = scene_pos[idxs]
 
-        for i in range(len(idxs)):
-            radius = 0.25
-            goal_color = mn.Color3(0, 153 / 255, 51 / 255)  # green
-            target_color = mn.Color4(1, 1, 1, 0.1)  # white, almost transparent
-            self._debug_line_render.draw_circle(
-                goal_pos[i], radius, goal_color
-            )
-            self._debug_line_render.draw_transformed_line(
-                target_pos[i], goal_pos[i], target_color, goal_color
-            )
-            # self._debug_line_render.draw_circle(target_pos[i], radius, goal_color)
+        if self._held_target_obj_idx != None:
+            color = mn.Color3(0, 255 / 255, 0)  # green
```

Comment:
FYI, I think there is a `mn.Color3.green()` https://doc.magnum.graphics/python/magnum/Color3/#green